### PR TITLE
Fix #490 make the Objective-C live view great again

### LINKED ISSUE #490: Objective-C live camera view doesn't work
The screen crashes on startup because the annotation view isn't initialized.
Once that is fixed the MLKit detection doesn't work because the capture session callback signature is wrong.

### PR DESCRIPTION
**Pull request by dustin-graham**
Fixes #490 
initializes the annotation view and corrects the capture session callback signature.

### CODE REVIEW

File: mlvision/MLVisionExampleObjc/CameraViewController.m [hunk 1]
```diff
@@ -282,30 +284,29 @@ - (CGPoint)normalizedPointFromVisionPoint:(FIRVisionPoint *)point
...
-- (void)captureOutput:(AVCaptureOutput *)output
-            didOutput:(CMSampleBufferRef)sampleBuffer
-       fromConnection:(AVCaptureConnection *)connection {
+- (void)captureOutput:(AVCaptureOutput *)output didOutputSampleBuffer:(CMSampleBufferRef)sampleBuffer fromConnection:(AVCaptureConnection *)connection {
   CVImageBufferRef imageBuffer = CMSampleBufferGetImageBuffer(sampleBuffer);
   if (imageBuffer) {
     FIRVisionImage *visionImage = [[FIRVisionImage alloc] initWithBuffer:sampleBuffer];
     FIRVisionImageMetadata *metadata = [[FIRVisionImageMetadata alloc] init];
-    UIImageOrientation orientation = [UIUtilities imageOrientationFromDevicePosition:_isUsingFrontCamera ? AVCaptureDevicePositionFront : AVCaptureDevicePositionBack];
+    UIImageOrientation orientation = [UIUtilities imageOrientationFromDevicePosition:AVCaptureDevicePositionBack];
```
[the remark this answers is not in the document]
Comment (dustin-graham) [reply mlvision/MLVisionExampleObjc/CameraViewController.m]: ah, that was debug code, forgot to revert. Thanks!

**Comment by dustin-graham** [comment]
@morganchen12 thanks for the review. I'm submitting my adjustments here in a few minutes. One question for you, perhaps you can help me solve a mystery. The ObjC target still has other defects that I found. For example, to toolbar at the top does not show. I could not figure out why, I spent about an hour trying different things. Would you mind taking a look to see if you can spot the problem? I'd really like to solve that mystery.

**Comment by morganchen12** [comment]
I'm able to see the toolbar at the top just fine. What device are you running on?

**Comment by dustin-graham** [comment]
It's an iPhone 6S running iOS 11.4

**Comment by dustin-graham** [comment]
What's the best way to test out the simulator? I can't load that screen on the simulator (no camera).

**Comment by morganchen12** [comment]
I see, I was looking at the wrong screen. I'll take a look at this tomorrow and get back to you.

**Pull request merged**